test_util/test_runner.py

```diff
@@ -150,18 +150,23 @@ def integration_test(
     try:
         with remote_port_forwarding(tunnel, agent_list+public_agent_list, join(test_dir, 'ssh_key')):
             log.info('Running integration test...')
-            tunnel.remote_cmd(test_cmd, timeout=timeout)
+            try:
+                tunnel.remote_cmd(test_cmd, timeout=timeout)
+            except CalledProcessError:
+                log.exception('Test failed!')
+                if ci_flags:
+                    return False
+                raise
         log.info('Successful test run!')
     except TimeoutExpired:
```

Comment:
TimeoutExpired Error must now be indented to be in order to catch the tunnel.remote_cmd(test_cmd, timeout=timeout)